source/configure/plugins-configuration-settings.rst

```diff
@@ -9,7 +9,6 @@ Self-hosted can manage the following configuration settings in **System Console
 - `Plugin Management <#plugin-management>`__
 - `Apps <#apps>`__
 - `Calls <#calls>`__
-- `Google Calendar <#google-calendar>`__
 - `MS Teams <#ms-teams>`__
```

Comment:
The MS Teams section is pretty big on this page. Maybe those settings could be put in another page like other integration plugins on the page?

Reply:
That's exactly the longer-term plan that will happen in a future release.